[
  {
    "cause": {
      "name": "SocketError",
      "code": "UND_ERR_SOCKET",
      "socket": {
        "localAddress": "192.168.1.101",
        "localPort": 39072,
        "remoteAddress": "190.217.24.69",
        "remotePort": 448,
        "remoteFamily": "IPv4",
        "bytesWritten": 1455,
        "bytesRead": 4366
      }
    },
    "llaveProceso": "11001400300120170130900",
    "iOfA": "4-120"
  }
]
